app/qml/form/MMFormPage.qml

```diff
@@ -281,7 +281,11 @@ Page {
         property var fieldWidget: model.EditorWidget
         property var fieldConfig: model.EditorWidgetConfig
 
-        property bool fieldIsReadOnly: root.state === "readOnly" || !AttributeEditable
+        // Following issue #3457, error message can be displayed even for non-editable attributes
+        // Thus, we split attributes indicating edit mode for formPage and attribute editability
+        property bool fieldIsReadOnly: root.state === "readOnly"
```

Comment:
Let's keep the naming consistency here and rename the `fieldIsReadOnly` to `fieldFormIsReadOnly`